### ISSUE TITLE
Drawer and Popup not working in production

### ISSUE THREAD

**Issue by annie444** [skeletonlabs/skeleton#1355]
### Current Behavior

So I've been using the dawer and popup components in dev and even in self-hosted production without much of a problem **on a computer**, however, neither of them seem to work on mobile.

Recently I've moved our productions app to Vercel, and now neither the drawer nor the popup are working. 

I've read through the [SSR docs](https://github.com/skeletonlabs/skeleton/wiki/SvelteKit-SSR-Warning) and applied the recommendations as well as the [sveltekit issue](https://github.com/sveltejs/kit/discussions/4339#discussioncomment-2384978) and tried everything to no avail. 

Any help would be appreciated

### Reproduction / Steps To Reproduce

[+layout.svelte](https://github.com/annie444/WardrobeDepot/blob/4ddf52bce69dbbc7032ffa54bbefadaba75ed694/src/routes/%2Blayout.svelte)
[NavBar.svelte](https://github.com/annie444/WardrobeDepot/blob/4ddf52bce69dbbc7032ffa54bbefadaba75ed694/src/routes/NavBar.svelte)
[NavTrail.svelte](https://github.com/annie444/WardrobeDepot/blob/4ddf52bce69dbbc7032ffa54bbefadaba75ed694/src/routes/NavTrail.svelte)

### Anything else?

If you want to clone my repo and take a look, make sure you include the necessary .env variables (dummy variables work just fine for purposes of reproducing this issue)

Take a look [here](https://github.com/annie444/WardrobeDepot/blob/4ddf52bce69dbbc7032ffa54bbefadaba75ed694/src/lib/config.server.ts) for the dummy schema

**Comment by endigo9740** [comment]
@annie444 we actually use Vercel for the Skeleton documentation. There's been no notable changes or regressions to these features that should cause these to stop working as described.

I see you're using the auto adapter in your project, which as far as I know is the recommendation when deploying to Vercel. We use it as well. Nothing is jumping out in your layout either, you do appear to be implementing these features as expected. The drawer data looks fine as well and Typescript should notify you if the data structure was out of whack.

> So I've been using the drawer and popup components in dev and even in self-hosted production without much of a problem on a computer, however, neither of them seem to work on mobile.

Can you expand on "not much of a problem". That implies there is -some- level of issue on desktop. Can you share if you have any particular console log errors as well?

Additionally I could see issues with popups if you're using the `hover` event type. Hover's not really a thing on mobile. In that case you would need to utilize click or focus instead if you wish to support mobile.

Again, you're welcome to review our documentation site and see these features running as expected. The doc site is always running the latest release of Skeleton, so it should reflect any issues that would affect users.

I'll tag our contributions @niktek @Sarenor  and @AdrianGonz97 to get a few more eyeballs here, but currently without a minimal reproduction of your issue, the issue may be something else at play on your end.

**Comment by annie444** [comment]
I actually was able to solve the problem by including `@skeletonlabs/skeleton` in the `optimizeDeps` section of my `vite.config.ts` and by implementing one of [these fixes](https://github.com/prisma/prisma/discussions/10767) for Prisma.

The Vercel logs showed an issue with the prisma bundle + sveltekit-superforms that prevented skeleton from loading fully. Then by including it in my `vite.config.ts` I was able to get a full reproduction using `npm run preview` on my local machine.

The earlier vagueness was because I was really only getting these issues on Vercel and not on my local machine.
